### DISCUSSION TITLE
Progress bar

### DISCUSSION THREAD

**Discussion by Windows1Door2Daniel** [FunkinCrew/Funkin#1659] [Ideas]
GG

can the game have a song progress bar to see your progress of the song

y'know like the youtube progress bar in a video

https://youtu.be/dQw4w9WgXcQ the progress bar in this video i mean

**Comment by Windows1Door2Daniel** [comment]
> Progress bar...
> @Windows1Door2Daniel Yes i like Progress bar
> ninjamuffin99: ok.

Yes i like Progress bar

**Comment by aflacc** [comment]
> > Progress bar...
> > @Windows1Door2Daniel Yes i like Progress bar
> > ninjamuffin99: ok.
> 
> Yes i like Progress bar

Yes i like Progress bar

**Comment by ACardboardBox1** [comment]
> 
> 
> Progress bar...
> @Windows1Door2Daniel Yes i like Progress bar
> ninjamuffin99: ok.

wtf

**Comment by GOOBOZA** [comment]
gometry dash. anyway it will have psych engine

**Comment by DotWith** [comment]
Yes i like Progress bar

**Reply by Windows1Door2Daniel** [reply]
Yes i like
![_20210826_112206](https://user-images.githubusercontent.com/81389527/131015875-480dfe93-959f-41c8-8037-05532e127895.jpg)

**Reply by DotWith** [reply]
I like "progress bar.."

**Comment by ninjamuffin99** [comment]
always on screen progress bar UI we think doesnt look good, so thats why we dont have it

**Comment by KitOdin** [comment]
i got fucking rickrolled lmao

**Reply by Windows1Door2Daniel** [reply]
Why

**Reply by KitOdin** [reply]
the youtube link i clicked.

**Reply by Windows1Door2Daniel** [reply]
Ok
